README.md

```diff
@@ -93,6 +93,27 @@ Example input and linked result:
 
 Also see [test cases](src/test/java/org/nibor/autolink/AutolinkUrlTest.java).
 
+### Simple link extraction
+
+Extract links not fully qualified URLs but still somewhat valid of the form `www.example.com`.
```

Comment:
Maybe: Extract links that don't start with "scheme://" but start with "www." such as `www.example.com`.